code/modules/mob/living/say.dm

```diff
@@ -93,6 +93,11 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	if(!message || message == "")
 		return
 
+	if(GLOB.in_character_filter.len && !forced)
+		if(findtext(message, config.ic_filter_regex))
+			to_chat(src, "<span class='warning'>That message contained a word prohibited in IC chat!</span>")

```

Comment:
this should probably be a configurable message as well

Reply:
People can already click the configurable rules button at the top of the game window, but I will alter the message to mention it.